components/action-mgt/org.wso2.carbon.identity.action.management/pom.xml

```diff
@@ -77,10 +77,18 @@
                         <Bundle-Name>${project.artifactId}</Bundle-Name>
                         <Private-Package>
                             org.wso2.carbon.identity.action.management.internal,
+                            org.wso2.carbon.identity.action.management.cache,
+                            org.wso2.carbon.identity.action.management.dao,
+                            org.wso2.carbon.identity.action.management.dao.impl,
+                            org.wso2.carbon.identity.action.management.util
                         </Private-Package>
                         <Export-Package>
                             !org.wso2.carbon.identity.action.management.internal,
-                            org.wso2.carbon.identity.action.management.*; version="${carbon.identity.package.export.version}"
+                            org.wso2.carbon.identity.action.management.*; version="${carbon.identity.package.export.version}",
+                            !org.wso2.carbon.identity.action.management.cache,
+                            !org.wso2.carbon.identity.action.management.dao,
+                            !org.wso2.carbon.identity.action.management.dao.impl,
+                            !org.wso2.carbon.identity.action.management.util
```

Comment:
Let's add these packages for the top for consistency

Reply:
This is updated with 9eaf3c0159310a8dde6d7706446804760e0a716a